ui/app/src/components/workspaces/WorkspaceUsers.tsx

```diff
@@ -28,9 +42,42 @@ export const WorkspaceUsers: React.FunctionComponent = () => {
     loadingState: LoadingState.Loading,
   });
 
+  const [selectedUserRole, setSelectedUserRole] = useState<IUser | undefined>(undefined);
+  const [hideCancelDialog, setHideCancelDialog] = useState(true);
+  const [deassigning, setDeassigning] = useState(false);
+  const [deassignmentError, setDeassignmentError] = useState(false);
+  const [apiError, setApiError] = useState({} as APIError);
+
+  const appRolesCtx = useContext(AppRolesContext);
+  const [isTreAdmin, setIsTreAdmin] = useState(false);
+
+  useEffect(() => {
+    setIsTreAdmin(appRolesCtx.roles.includes('TREAdmin'));
+  }, [appRolesCtx.roles]);
+
+  const navigate = useNavigate();
+  const theme = getTheme();
   const apiCall = useAuthApiCall();
-  const { workspace, roles, workspaceApplicationIdURI } =
-    useContext(WorkspaceContext);
+  const { workspace, roles, workspaceApplicationIdURI } = useContext(WorkspaceContext);
+
+  const [loadingUsers, setloadingUsers] = useState(false);
+
+  const isTemplateVersionValid = (): Boolean => {
+    const templateVersion = workspace.templateVersion;
+    const templateElements = templateVersion.split('.');
+    const major = parseInt(templateElements[0]);
+    const minor = parseInt(templateElements[1]);
+
+    // Base template version 2.1.0 is the minimum required
+    return (major > 2 || (major === 2 && minor >= 1));
+  }
+
+  const allowUserManagement = useMemo(() => {
```

Comment:
So, do we simply not show the UI? Is there anything to help someone work out "why isn't the UI showing"?

Reply:
I have added a note to the changelog to detail what is needed to get the UI to show, will take a look and see if there is anywhere else in the docs it is needed.